server.js

```diff
@@ -1,14 +1,22 @@
-const { ApolloServer } = require('apollo-server');
+const { ApolloServer, AuthenticationError } = require('apollo-server');
 const typeDefs = require('./typeDefs');
 const resolvers = require('./resolvers');
 const connectDB = require('./config/db');
 
+const verifyUser = require('./utils/verifyUser');
+
 connectDB();
 require('dotenv').config();
 
 const server = new ApolloServer({
   typeDefs,
   resolvers,
+  context: ({ req }) => {
+    const { token } = req.headers || req.cookies || '';
```

Comment:
Does it work? We should use the header with the name `Authorization` to get the bearer jwt token. Why do we need `req.cookies` here?

Reply:
req.headers is work, req.cookies was deleted